ADR/ADR044-translate-api-response-at-request-time.md

```diff
@@ -0,0 +1,336 @@
+# ADR044: Translate API response at request time
+
+Date: 2025-08-01
+
+## Status
+
+Accepted
+## Context
+
```

Comment:
Do we need to reference https://github.com/alphagov/forms/blob/main/ADR/ADR007-store-form-text-data-as-objects.md

Which was an early, similar ADR which has remained at proposed.

Obviously things have moved on so much from then we could just dismiss it?

Reply:
Good spot - I'd forgotten that ADR existed! Have updated to supersede that ADR, clarify that it was never implemented, and added a short note to this ADR about it as well.